tools/pkglint/lib/rules.ts

```diff
@@ -776,7 +776,7 @@ export class MustHaveIntegCommand extends ValidationRule {
     expectDevDependency(this.name,
       pkg,
       'cdk-integ-tools',
-      `file:${path.relative(pkg.packageRoot, path.resolve(__dirname, '../../cdk-integ-tools'))}`);
+      `^${pkg.json.version}`);
```

Comment:
This actually needs to be repo global version, not pkg.version, no? Wouldn't we want pkglint to discover if there's a mismatch (for some reason, e.g. a user created a new module and put the wrong version there).

Reply:
They should be the same... `bump.sh` will fix them up to the same thing (repo-global & package version). I could change it there, but I feel it'd divert from the current mode of operation where it only looks at a single package... where now it'd look at files at a know fixed location *outside* of the package...